tools/osx-pkg-postinstall.sh

```diff
@@ -3,3 +3,4 @@
 # TODO Can we extract $PREFIX from the installer?
 cd /usr/local/bin
 ln -sf ../lib/node_modules/npm/bin/npm-cli.js npm
+ln -sf ../lib/node_modules/node-inspect/cli.js node-inspect
```

Comment:
It was the only "obvious" part that seemed to expose the binaries. I wasn't sure how to test the installer w/o the signing keys. :( Should I revert this change and declare "integrate with OSX/windows installer" out of scope for this PR?